solidity/contracts/bridge/Bridge.sol

```diff
@@ -15,49 +15,33 @@
 
 pragma solidity 0.8.4;
 
+import "./BitcoinTx.sol";
+
 import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
 import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
 
-/// @title BTC Bridge
-/// @notice Bridge manages BTC deposit and redemption and is increasing and
+/// @title Bitcoin Bridge
+/// @notice Bridge manages BTC deposit and redemption flow and is increasing and
 ///         decreasing balances in the Bank as a result of BTC deposit and
-///         redemption operations.
+///         redemption operations performed by depositors and redeemers.
 ///
-///         Depositors send BTC funds to the most-recently-created-wallet of the
-///         bridge using pay-to-script-hash (P2SH) or
-///         pay-to-witness-script-hash (P2WSH) which contains hashed
-///         information about the depositor’s minting Ethereum address. Then,
-///         the depositor reveals their desired Ethereum minting address to the
-///         Ethereum chain. The Bridge listens for these sorts of messages and
-///         when it gets one, it checks the Bitcoin network to make sure the
-///         funds line up. If they do, the off-chain wallet may decide to pick
-///         this transaction for sweeping, and when the sweep operation is
-///         confirmed on the Bitcoin network, the wallet informs the Bridge
-///         about the sweep increasing appropriate balances in the Bank.
+///         Depositors send BTC funds to the most recently created off-chain
+///         ECDSA wallet of the bridge using pay-to-script-hash (P2SH) or
+///         pay-to-witness-script-hash (P2WSH) containing hashed information
+///         about the depositor’s Ethereum address. Then, the depositor reveals
+///         their desired Ethereum address along with their deposit blinding
+///         factor to the Bridge on Ethereum chain. The off-chain ECDSA wallet
```

Comment:
Actually, the depositor needs to reveal also the refund public key hash and the refund locktime. Also, the wallet public key hash used during deposit creation should be revealed but that can be inferred based on most recently created wallet

Reply:
See https://github.com/keep-network/tbtc-v2/pull/107/commits/46cee25ccb48836534f2189049db3b29482d6fd2, please.